### PR TITLE
Updating show model

### PR DESCRIPTION
**Pull request by Lunalo**
in progress

@stevenndungu  You can now close the related branch

### CODE REVIEW

**Comment by Lunalo** [comment]
@dannyparsons 

I have small problem with fixing the resetting the ucrchkEnterValues  and  saving on grid(I have tried to fix this but is returning output both in the output window and the grid)

**Comment by dannyparsons** [comment]
Ok I can check this out, but do you know why there are conflicts here?

**Comment by Lunalo** [comment]
yes. Steve is supposed the branch he is working on

**Comment by Lunalo** [comment]
@dannyparsons 

I am making final fix on this

**Comment by Lunalo** [comment]
@dannyparsons 

I have now fixed everything else except the ressetting of the checkbox of the ucrsave

**Comment by dannyparsons** [comment]
What do you suggest here about the conflicts?

**Comment by Lunalo** [comment]
create another branch

File: instat/dlgShowModel.vb [hunk 1]
```diff
@@ -43,186 +63,193 @@ Public Class dlgShowModel
...
+        ucrDistributionsFOrTablePlus.SetParameters()
+
+        ucrChkEnterValues.SetText("Enter value(s)")
+        ucrChkDisplayGraphResults.SetParameter(New RParameter("plot", 2), bNewChangeParameterValue:=True, bNewAddRemoveParameter:=True, strNewValueIfChecked:="TRUE", strNewValueIfUnchecked:="FALSE")
+        ucrChkDisplayGraphResults.SetRDefault("TRUE")
+        ucrChkDisplayGraphResults.SetText("Display Graph Results")
+
+        ucrChkEnterValues.AddToLinkedControls(ucrReceiverExpressionForTablePlus, {False}, bNewLinkedHideIfParameterMissing:=True)
+        ucrChkEnterValues.AddToLinkedControls(ucrInputProbabilities, {True}, bNewLinkedHideIfParameterMissing:=True, objNewDefaultState:=0.5)
+        ucrChkEnterValues.AddParameterIsRFunctionCondition(False, "q", True)
```
Comment (lilyclements) [inline instat/dlgShowModel.vb]: Out of interest, what is `.AddParameterIsRFunctionCondition`? I don't think I've come across this before?

Comment (dannyparsons) [inline instat/dlgShowModel.vb]: this conditions checks if the value of the parameter is an RFunction. For cases when the easiest way to check between two cases is to see if the parameter value is a string or an RFunction e.g. a column from the data.

**Comment by stevenndungu** [comment]
@dannyparsons  This can be closed now, replaced by pull request #3546

**Pull request closed without merging**